src/app/space/create/deployments/apps/deployment-card.component.html

```diff
@@ -1,31 +1,42 @@
 <div class="card-pf" (click)="collapsed = !collapsed">
   <div class="card-pf-body">
-    <span class="pficon pficon-ok" tooltip="Everything looks okay" placement="top"></span>
-    <deployments-donut [applicationId]="applicationId" [mini]="true" ></deployments-donut>
-    <span id="versionLabel">{{ version | async }}</span>
-    <div class="pull-right dropdown-kebab-pf dropdown" dropdown (click)="$event.stopPropagation()">
-      <button class="btn btn-link pull-right dropdown-toggle" id="kebab" type="button" aria-haspopup="true"
-        aria-expanded="true" data-toggle="dropdown" dropdownToggle>
-        <span class="fa fa-ellipsis-v"></span>
-      </button>
-      <ul class="dropdown-menu-right dropdown-menu" aria-labelledby="kebab" role="menu" *dropdownMenu>
-        <li>
-          <a>View logs</a>
-        </li>
-        <li>
-          <a>View OpenShift Console</a>
-        </li>
-        <li role="separator" class="divider"></li>
-        <li>
-          <a>Open Application</a>
-        </li>
-        <li role="separator" class="divider"></li>
-        <li>
-          <a>Delete</a>
-        </li>
-      </ul>
+    <div class="row mini-card-content">
+      <div class="col-xs-2">
+        <span class="pficon fa-lg pficon-ok" tooltip="Everything looks okay" placement="top"></span>
```

Comment:
write the icon classes using the order 1. family 2. icon name 3. size: `pficon pficon-ok fa-lg`